ci/diff.py

```diff
@@ -1,12 +1,39 @@
 import difflib
 import json
+import re
 import sys
+from typing import Any, Dict, List
 
 
-def group_objects(json_file_path):
+def replace_timestamps(obj: Any) -> Any:
```

Comment:
```suggestion
from typing import TypeVar

TSType = TypeVar("TSType", str, list, dict)

def replace_timestamps(obj: TSType) -> TSType:
```

This is not perfect, but provides some very rudimentary type checking:

```py
reveal_type(replace_timestamps("2020-01-01T00:00:00.000Z"))
reveal_type(replace_timestamps(["2020-01-01T00:00:00.000Z", "2020-01-01T00:00:00.000Z"]))
reveal_type(replace_timestamps({"foo": "2020-01-01T00:00:00.000Z"}))
```
```
% mypy ci
ci/diff.py:27: note: Revealed type is "builtins.str"
ci/diff.py:28: note: Revealed type is "builtins.list[Any]"
ci/diff.py:29: note: Revealed type is "builtins.dict[Any, Any]"
```

With a more complicated `TypeVar` we could remove the `Any` from the returned `list` and `dict` type annotations, but that's overkill for something that isn't even part of the CLI itself.